View/DefaultTranslatedView.php

```diff
@@ -2,6 +2,11 @@
 
 namespace BabDev\PagerfantaBundle\View;
 
+@trigger_error(sprintf('The "%s" class is deprecated and will be removed in BabDevPagerfantaBundle 3.0. Use the "%s" class instead.', DefaultTranslatedView::class, TwigView::class));
```

Comment:
Why not use `__CLASS__` here?

Reply:
The trigger is outside the class definition, so the magic constant won't work.

Reply:
Right, I missed that.